Array_contains process

zansinergise opened the pull request:
https://git.sinergise.com/team-6/openeo-platform/-/issues/63

dgostencnik commented on tests/unit_tests/test_array_contains.py:
```diff
@@ -0,0 +1,30 @@
+        ({'data': [1,2,3], 'value': 2}, True),
+        ({'data': ["A","B","C"], 'value': "b"}, False),
+        ({'data': [1,2,3], 'value': "2"}, False),
+        ({'data': [1,2,None], 'value': None}, False),
```
This is not in line with [Example 4](https://docs.openeo.cloud/processes/#array_contains), however it seems that either example 4 or operation definition is wrong (last bullet).

zansinergise replied:
I went with the assumption that the definition is correct

dgostencnik commented:
additional cases that should be handled (red asterisk marks mandatory argument) 
 - data is undefined ({"value": "2"}, raise_ex),
 - data is null ({"data": None, "value": "2"}, raise_ex),
 - value is undefined ({"data": [1, 2, 3]}, raise_ex),

zansinergise commented:
Fixed the process accordingly and added these test cases

zansinergise commented:
> additional cases that should be handled (red asterisk marks mandatory argument)
> 
> * data is undefined ({"value": "2"}, raise_ex),
> * data is null ({"data": None, "value": "2"}, raise_ex),
> * value is undefined ({"data": [1, 2, 3]}, raise_ex),

Fixed the process accordingly and added these test cases
